agent/acs/client/acs_client_test.go

```diff
@@ -18,6 +18,7 @@ package acsclient
 import (
 	"encoding/json"
 	"errors"
+	"github.com/aws/amazon-ecs-agent/agent/api/eni"
```

Comment:
can you organize the import following the guideline [here](https://github.com/golang/go/wiki/CodeReviewComments#imports)? same for other imports you added